pkg/controller/instance/instance_controller.go

```diff
@@ -536,5 +563,8 @@ func scheduledPlan(i *kudoapi.Instance, ov *kudoapi.OperatorVersion) (string, ty
 		i.Spec.PlanExecution.Status = kudoapi.ExecutionNeverRun
 	}
 
+	if i.Spec.PlanExecution.PlanName == kudoapi.DeployPlanName || i.Spec.PlanExecution.PlanName == kudoapi.UpgradePlanName || i.Spec.PlanExecution.PlanName == kudoapi.UpdatePlanName {
```

Comment:
I'm not sure if we use these things somewhere else, but I wouldn't mind something like `kudoapi.isDefaultPlan(planName)` or similar...

Reply:
yeah Aleksey wanted that as well I just don't know how to name it ... I am not sure default plan means something to me really. I don't want to do "IsDeployOrUpgradeOrUpdate" 🤷  so I ended up keeping this explicitly :/